index.html

```diff
@@ -17,10 +17,52 @@
   </head>
   <body>
     <header>
-      <div class="container"></div>
+      <div class="container">
+        <div class="row">
+          <div class="column-full">
+            <h1 class="heading">Code Journal</h1>
+          </div>
+        </div>
+      </div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
+      <form class="column-full">
```

Comment:
This isn't inside of a `row` so `column-full` isn't actually doing anything, so it should be removed